# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/Lowering.h
# Commit: ecb8f8821de2
# Full Hash: ecb8f8821de2ab318eb4a0d863b5d9c1cd97d70f
# Author: Tooru Fujisawa <arai_a@mac.com>
# Date: 2016-04-07 20:42:20
# Regressor Bug: 1165052
# File Overlap Count: 1
# Description:
#   Bug 1165052 - Part 4: Inline IsConstructor intrinsic. r=jandem
# ==============================================================================

diff -r ae46f7e09a22 -r ecb8f8821de2 js/src/jit/Lowering.h
--- a/js/src/jit/Lowering.h	Wed Dec 23 12:46:35 2015 +0900
+++ b/js/src/jit/Lowering.h	Mon Nov 16 09:01:37 2015 +0900
@@ -271,6 +271,7 @@
     void visitInstanceOf(MInstanceOf* ins);
     void visitCallInstanceOf(MCallInstanceOf* ins);
     void visitIsCallable(MIsCallable* ins);
+    void visitIsConstructor(MIsConstructor* ins);
     void visitIsObject(MIsObject* ins);
     void visitHasClass(MHasClass* ins);
     void visitAsmJSLoadGlobalVar(MAsmJSLoadGlobalVar* ins);